modules/tensor_mechanics/src/materials/ADSingleVariableReturnMappingSolution.C

```diff
@@ -193,7 +199,7 @@ ADSingleVariableReturnMappingSolution::internalSolve(const ADReal effective_tria
                             max_permissible_scalar,
                             iter_output);
 
-    _residual = computeResidual(effective_trial_stress, scalar);
+    computeResidualAndDerivativeHelper(effective_trial_stress, scalar);
```

Comment:
Is the derivative required here?

Reply:
No, good catch.

Reply:
Oh wait. Here it is required, it will be used in line 191 the next iteration in this while loop.